Aquality.Selenium/tests/Aquality.Selenium.Tests/Resources/settings.json

```diff
@@ -19,6 +19,7 @@
         "download.prompt_for_download": "false",
         "download.default_directory": "./downloads"
       },
+      "excludedArguments": [ "enable-automation" ],
```

Comment:
I believe we use a different settings for azure pipelines: settings.azure.json

Reply:
Add this into settings.azure.json ? or what do you mean?

Reply:
done